packages/core/src/rum/instrumentation/resourceTracking/implementation/DatadogRumResource/internalDevResourceBlocklist.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import type { RUMResource } from '../../domain/interfaces/RumResource';
+
+/**
+ * Expo sends all console.* calls to the packager. As we log all API calls
+ * when the SDKVerbosity is DEBUG, this would result in an infinite loop of
+ * `console.log` and API calls, creating 250 RUM resources per second.
+ *
+ * The hostname is always going to be localhost or a local IP.
+ *
+ * An example URL is http://192.168.1.20:8081/logs or http://10.46.29.155:19000/logs
+ */
+const EXPO_DEV_LOGS_REGEX = new RegExp(
+    '^http://((10|172|192).[0-9]+.[0-9]+.[0-9]+|localhost):[0-9]+/logs$'
+);
+
+/**
+ * This call is made every time the RN packager reloads the js in dev mode.
+ */
+const RN_PACKAGER_SYMBOLICATE_REGEX = new RegExp(
```

Comment:
is this also related with `EXPO` ?

Reply:
This happens in Expo and RN but because Expo uses RN. 
I see it a lot in my logs, so I thought it would be good to get rid of it as well.

Reply:
This does not trigger an infinite loop, but I think it is likely to represent a significant amount of resources across all users